src/components/colcardlist.js

```diff
@@ -14,13 +16,13 @@ function ColCardList() {
         />
         <Card
           type="col"
-          src={icon_1}
+          src={icon_6}
           header="Suitable for ads, email subject lines, blogposts & landing page headlines"
           content="All headlines are filtered by platform. You can use them for advertising, landing pages, emails or blogs posts."
         />
         <Card
           type="col"
-          src={icon_1}
+          src={icon_7}
```

Comment:
Prefer using camelCase + non-digit in naming. This one can be turned to `iconSeven`